api/python/quilt3/packages.py

```diff
@@ -1229,6 +1231,15 @@ def physical_key_is_temp_file(pk):
             self._set(lk, pkg[lk])
 
         pkg._build(name, registry=registry, message=message)
+
+        navigator_url = get_from_config("navigator_url")
+        registry_parsed = PhysicalKey.from_url(registry)
+        shorthash = Package._shorten_tophash(name, registry_parsed, pkg.top_hash)
+        print(f"Package {name}@{shorthash} pushed to s3://{dest_parsed.bucket}")
+        if navigator_url is None:
+            print(f"Run `quilt3 catalog {str(dest_parsed)}` to browse.")
+        else:
+            print(f"Visit {navigator_url}/b/{dest_parsed.bucket}/packages/{name}/tree/latest")
```

Comment:
I'd suggest moving this logic to `util.py`, either in `catalog_s3_url` or create a new method `catalog_package_url`.